ops/ansible/roles/bfd-server/templates/newrelic.yml.j2

```diff
@@ -0,0 +1,18 @@
+# This file configures the New Relic Agent.  New Relic monitors
+# Java applications with deep visibility and low overhead.  For more details and additional
+# configuration options visit https://docs.newrelic.com/docs/agents/java-agent/configuration/java-agent-configuration-config-file.
+
+transaction_events:
```

Comment:
We'll want to add the async headers as well, namely the bulk headers:

```
* `BULK-CLIENTID`: a unique identifier of the bulk client for whom this request is for (i.e. UUID, ACO ID, NPI, Part D Contract)
* `BULK-JOBID`: a unique identifier for the job (Job ID, UUID, etc.)
```

Reply:
Added!